zone/guild_mgr.cpp

```diff
@@ -396,69 +411,85 @@ bool ZoneGuildManager::VerifyAndClearInvite(uint32 char_id, uint32 guild_id, uin
+{
+	switch (pack->opcode)
+	{
+	case ServerOP_RefreshGuild:
+	{
+		if (pack->size != sizeof(ServerGuildRefresh_Struct)) {
 			LogError("Received ServerOP_RefreshGuild of incorrect size [{}], expected [{}]", pack->size, sizeof(ServerGuildRefresh_Struct));
 			return;
 		}
-		ServerGuildRefresh_Struct *s = (ServerGuildRefresh_Struct *) pack->pBuffer;
 
-		LogGuilds("Received guild refresh from world for [{}], changes: name=[{}], motd=[{}], rank=[{}], relation=[{}]", s->guild_id, s->name_change, s->motd_change, s->rank_change, s->relation_change);
+		ServerGuildRefresh_Struct* s = (ServerGuildRefresh_Struct*)pack->pBuffer;
+		LogDebug("Received guild refresh from world for guild id [{}] changes: name=[{}] motd=[{}] rank=[{}] relation=[{}]",
+			s->guild_id,
+			s->name_change,
+			s->motd_change,
+			s->rank_change,
+			s->relation_change
+		);
 
-		//reload all the guild details from the database.
 		RefreshGuild(s->guild_id);
 
-		if(s->motd_change) {
+		if (s->motd_change) {
 			//resend guild MOTD to all guild members in this zone.
 			entity_list.SendGuildMOTD(s->guild_id);
 		}
 
-		if(s->name_change) {
+		if (s->name_change) {
 			//until we figure out the guild update packet, we resend the whole guild list.
 			entity_list.SendGuildList();
 		}
 
-		if(s->rank_change) {
+		if (s->rank_change) {
 			//we need to send spawn appearance packets for all members of this guild in the zone, to everybody.
 			entity_list.SendGuildSpawnAppearance(s->guild_id);
 		}
 
-		if(s->relation_change) {
+		if (s->relation_change) {
 			//unknown until we implement guild relations.
 		}
 
 		break;
 	}
 
-	case ServerOP_GuildCharRefresh: {
-		if(pack->size != sizeof(ServerGuildCharRefresh_Struct)) {
+	case ServerOP_GuildCharRefresh:
+	{
+		if (pack->size != sizeof(ServerGuildCharRefresh_Struct)) {
 			LogError("Received ServerOP_RefreshGuild of incorrect size [{}], expected [{}]", pack->size, sizeof(ServerGuildCharRefresh_Struct));
 			return;
 		}
-		ServerGuildCharRefresh_Struct *s = (ServerGuildCharRefresh_Struct *) pack->pBuffer;
-
-		LogGuilds("Received guild member refresh from world for char [{}] from guild [{}]", s->char_id, s->guild_id);
+		ServerGuildCharRefresh_Struct* s = (ServerGuildCharRefresh_Struct*)pack->pBuffer;
+		LogDebug("Received guild member refresh from world for char [{}] from guild [{}]",
+			s->char_id,
+			s->guild_id
+		);
+		RefreshGuild(s->guild_id);
 
-		Client *c = entity_list.GetClientByCharID(s->char_id);
+		Client* c = entity_list.GetClientByCharID(s->char_id);
 
-		if(c != nullptr) {
+		if (c != nullptr) {
 			//this reloads the char's guild info from the database and sends appearance updates
 			c->RefreshGuildInfo();
 		}
 
 		//it would be nice if we had the packet to send just a one-person update
-		if(s->guild_id == GUILD_NONE) {
-			if(c != nullptr)
-				c->SendGuildMembers();	//only need to update this player's list (trying to clear it)
-		} else {
+		if (s->guild_id == GUILD_NONE) {
+			if (c != nullptr) {
```

Comment:
`if (!c)` does the same thing

Reply:
Updated several occurrences of this in next commit.